bin/alluxio

```diff
@@ -40,6 +43,21 @@ fi
 COMMAND=$1
 shift
 
+ALLUXIO_SHELL_JAVA_OPTS=""
+while getopts ":D:" opt; do
+  case $opt in
+  D)
+    ALLUXIO_SHELL_JAVA_OPTS+=" -D$OPTARG"
+    ;;
+  \?)
+    echo "Invalid option: -$OPTARG"
+    printUsage
+    exit 1
+    ;;
+  esac
+done
+shift $(expr $OPTIND - 1 )
```

Comment:
nit: shift $((${OPTIND} - 1)) is preferred.

Per google style guide:
Prefer this:
addition=$((${X} + ${Y}))
substitution="${string/#foo/bar}"